package.json

```diff
@@ -3,11 +3,12 @@
   "version": "1.0.0",
   "description": "Basic Structure for React app following Atomic Design",
   "scripts": {
-    "start": "webpack-dev-server --inline --hot --progress --colors",
-    "build": "webpack --config ./webpack-production.config.js --progress --colors",
-    "lint": "eslint ./app/ --ext .js --ext .jsx --fix",
-    "watch": "webpack --watch --progress",
-    "clean": "rm -rf dist && npm cache clean"
+    "start":    "webpack-dev-server --inline --hot --progress --colors",
+    "prebuild": "npm run clean",
+    "build":    "webpack --config ./webpack-production.config.js --progress --colors",
+    "lint":     "eslint ./app/ --ext .js --ext .jsx --fix",
+    "watch":    "webpack --watch --progress",
+    "clean":    "rm -rf dist && npm cache clean"
```

Comment:
Any good reason why we would want to clean the cache on every pre build? @Oliphaunte